### DISCUSSION TITLE
Content Localization on FrontEnd

### DISCUSSION THREAD

**Discussion by aashish803** [OrchardCMS/OrchardCore#16486] [General]
Hey 

I want to add the localization to my orchard core website and I followed the following process to achieve that functionality:-

1. Enable features like " Localization", " Content Culture Picker " and  " Content Localization "
2. Create a layer with the culture condition 
3. Create a content type in which I want the localization to appear and add the localization part
4. Create some content Items of that content type and add the shape  {% shape "ContentCulturePicker" %} in my content type's template. 

But after following the above procedure I am getting the runtimebinding error 
![image](https://github.com/user-attachments/assets/614f79be-3974-45c9-8773-b796f87e1db4)

Can you please tell me what I am doing wrong so that achieve the localization in my site?

Thanks

**Comment by hishamco** [comment]
@MikeAlhayek I remembered no need to add shape because you add it to navbar after enabling content localization, right?

**Reply by aashish803** [reply]
@hishamco I have added it to the content type's template.

**Reply by hishamco** [reply]
Could you please check the last two release notes I am sure there's something related to the changes in content localization picker

**Reply by aashish803** [reply]
I have checked the change logs of previous version and  latest version, and I have not found anything. Would you share the URL or something for me to read?

**Comment by gaojianzhuang** [comment]
@aashish803  @hishamco  any update? Thanks.

**Comment by hishamco** [comment]
Is this issue still valid?

**Comment by gaojianzhuang** [comment]
Found another way to implement the localization features. The codes as below:
```
{% assign currentCulture = Culture.Name %}
{% assign cultures = Culture | supported_cultures %}
<div class="culture-picker dropdown">
	<button class="btn btn-success btn-sm dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">{{currentCulture}}</button>
	<ul class="dropdown-menu dropdown-menu-end shadow">
		{% for culture in cultures %}
			{% if culture.Name != currentCulture %}
			<li><a class="dropdown-item" href="{{ culture.Name | switch_culture_url }}">{{ culture.Name }}</a></li>
			{% else %}
			<li><a class="dropdown-item active" href="{{ culture.Name | switch_culture_url }}">{{ culture.Name }}</a></li>
			{% endif %}
		{% endfor %}
	</ul>
</div>
```
You can add these codes in the layout.liquid file or create a Layout template data.
